A76-ring-hash-improvements.md

```diff
@@ -178,6 +186,10 @@ endpoint attribute to the value of [LbEndpoint.Metadata][LbEndpoint.Metadata]
 `envoy.lb` `hash_key` field, as described in [Envoy's documentation for the ring
 hash load balancer][envoy-ringhash].
 
+If multiple endpoints have the same `hash_key`, then which endpoint is chosen is
```

Comment:
I will note that this problem is not strictly new in this gRFC, because it's conceivably possible for there to be hash collisions even in the original design.  But I agree that the fact that we're allowing hash keys to be specified manually makes it more likely, so I'm fine with noting this here.

Reply:
done.